backend/config/settings/base.py

```diff
@@ -120,6 +122,29 @@
 # https://docs.djangoproject.com/en/4.1/howto/static-files/
 
 STATIC_URL = "static/"
+STATIC_ROOT = os.path.join(BASE_DIR, "staticfiles")
+
```

Comment:
Add compression and caching support for staticfiles:
`STATICFILES_STORAGE = "whitenoise.storage.CompressedManifestStaticFilesStorage"`